src/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-grow: 1;
+  flex-basis: 300px;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  background-color: red;
+}
+
+/*#region Background*/
+.block-2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.block-3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.block-4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.block-5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.block-6 {
+  background-color: black;
+}
+/*#endregion*/
+
+/*#region mediaWidth*/
+@media (min-width:600px) and (max-width:750px) {
```

Comment:
that what i see between 750 and 900 px
![image](https://user-images.githubusercontent.com/37161315/167660579-be205cda-5ee6-4a6d-a84b-30559ff5f4ba.png)
remove this max-width par of media rule, why do you need it?
```suggestion
@media (min-width:600px) {
```